flink/v1.12/flink/src/main/java/com/netease/arctic/flink/util/ArcticUtils.java

```diff
@@ -177,17 +196,58 @@ public static ArcticLogWriter buildArcticLogWriter(Map<String, String> propertie
         "'. only support 'v1' or empty");
   }
 
-  public static <T> LogMsgFactory<T> buildLogMsgFactory(Map<String, String> tableProperties) {
+  private static Properties combineArcticProperties(Map<String, String> tableProperties,
```

Comment:
Judging by the name of this method `combineArcticProperties`, the function of this method is not very clear to devs. Maybe `combineLogstoreQueueProperties` is better. WDYT?